Sources/BraveOnboarding/OnboardingPreferences.swift

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import BraveShared
+import struct Shared.Strings
+import BraveCore
+
+extension Preferences {
+  public final class General {
```

Comment:
This may cause ambiguity since `ClientPreferences` already defines it.  We can create a new name specifically for this module i.e. `Preferences.Onboarding`.  The keys for the containing options can remain the same

Reply:
Fixed at https://github.com/brave/brave-ios/pull/6270/commits/1b5bbd88bd88e295e76cecf0c75da4fe09932d17